en/docs/basics_topics/model_load_save.md

```diff
@@ -240,42 +237,34 @@ lenet_models_name
 
 We can see:
 
-* The job function in network, each of the variable have a sub-directory.
+* The network in job function, each variable is corresponding to a sub-directory.
 
-* All subdirectories above have an  `out` document. It store the parameters of network in binary. `Out` is the default file name. We can change that by  `variable op` in the network.
+* All subdirectories above have an  `out` document. It store the parameters of network in binary form. `Out` is the default file name. We can change that by  `variable op` in the network.
 
-* `Snapshot_done` is an empty folder. If it exists, means the network is completed the training.
+* `Snapshot_done` is an empty folder. If it exists, it means that the network has been trained. 
 
-* Snapshots of the training steps is store in `System-Train-TrainStep-train_job`.
+* Snapshots of the training steps is stored in `System-Train-TrainStep-train_job`.
 
 ## Model finetune and transfer learning
 
-We will meet the following scenario when we do the model finetune and transfer learning:
-
-* Finetune: Use a backbone which already completed the training as foundation. Keep training after expand some new network structure. The model of original backbone network need load the parameters we store previously. But the new part of the network need initialization.
-
-* Transfer learning: We will training according to new optimization method base on the original network which have completed the training. The new optimization method bring some extra parameters. Such as `momentum` or `adam`. The original parameters need load the parameters we store previously. But the extra parameters need initialization.
-
-
-To summarise, the above situation is:
-
-* Part of the parameters in model are load from original model.
+In model finetune and transfer learning, we always need：
 
-* The other part(new) of parameters need initialization.
+- Load some of the parameters from original model
+- Initialize the other part of parameters in model 
 
-For this, `flow.train.CheckPoint.load` of OneFlow set the following procedures:
+For this, the following procedures in `flow.train.CheckPoint.load` are as follow:
 
-* According the description of network model in job function. The saving path of ergodic model and try to load each parameters.
+* According the model in job function, traverse the saved path of the model and try to load the parameters.
```

Comment:
According the model in job function -> According to the model defined in job function ...